source/_integrations/zamg.markdown

```diff
@@ -18,7 +18,7 @@ ha_codeowners:
 
 The `zamg` platform uses meteorological details published by the Austrian weather service [Zentralanstalt für Meteorologie und Geodynamik (ZAMG)](https://www.zamg.ac.at).
 
-Only observations for capital cities are publicly available. You can check the list of stations in [CSV format](https://www.zamg.ac.at/ogd).
+With this integration it is possible to get weather station data from up to 228 different zamg stations across Austria. The integration is using the [datahub-api](https://dataset.api.hub.zamg.ac.at/v1/docs/index.html). The old `odg` api (before 2022.11.1) was only able to read from 22 mayor stations.
```

Comment:
We don't have to keep a changelog here. Let's remove "the old" part.

Reply:
ok, done